### PR TITLE
troubleshooting option to check GPO

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5726

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

File: windows/deployment/update/waas-delivery-optimization.md [hunk 1]
```diff
@@ -223,6 +223,15 @@ If you suspect this is the problem, try a Telnet test between two devices on the
 2. Run the test. For example, if you are on device with IP 192.168.8.12 and you are trying to test the connection to 192.168.9.17 run **telnet 192.168.9.17 7680** (the syntax is *telnet [destination IP] [port]*. You will either see a connection error or a blinking cursor like this /_. The blinking cursor means success.
 
 
+### None of the computers on the network are peering
+
+If you suspect this is the problem, check Delivery Optimization settings that could limit participation in Peer Caching. Check following settings in assigned group policies, local group policies and MDM policies, whether they are too restrictive:
```
Comment (JohanFreelancer9) [inline windows/deployment/update/waas-delivery-optimization.md:228]:
```suggestion
If you suspect this is the problem, check Delivery Optimization settings that could limit participation in Peer Caching. Check whether the following settings in assigned group policies, local group policies, and MDM policies are too restrictive:
```

File: windows/deployment/update/waas-delivery-optimization.md [hunk 2]
```diff
@@ -223,6 +223,15 @@ If you suspect this is the problem, try a Telnet test between two devices on the
...
+
+- Minimum RAM (inclusive) allowed to use Peer Caching
```
Comment (JohanFreelancer9) [inline windows/deployment/update/waas-delivery-optimization.md:230]:
```suggestion
- Minimum RAM (inclusive) allowed to use Peer Caching.
```

File: windows/deployment/update/waas-delivery-optimization.md [hunk 3]
```diff
@@ -223,6 +223,15 @@ If you suspect this is the problem, try a Telnet test between two devices on the
...
+- Minimum disk size allowed to use Peer Caching
```
Comment (JohanFreelancer9) [inline windows/deployment/update/waas-delivery-optimization.md:231]:
```suggestion
- Minimum disk size allowed to use Peer Caching.
```

File: windows/deployment/update/waas-delivery-optimization.md [hunk 4]
```diff
@@ -223,6 +223,15 @@ If you suspect this is the problem, try a Telnet test between two devices on the
...
+- Enable Peer Caching while the device connects via VPN
```
Comment (JohanFreelancer9) [inline windows/deployment/update/waas-delivery-optimization.md:232]:
```suggestion
- Enable Peer Caching while the device connects via VPN.
```

File: windows/deployment/update/waas-delivery-optimization.md [hunk 5]
```diff
@@ -223,6 +223,15 @@ If you suspect this is the problem, try a Telnet test between two devices on the
...
+- Allow uploads while the device is on battery while under set Battery level
```
Comment (JohanFreelancer9) [inline windows/deployment/update/waas-delivery-optimization.md:233]:
```suggestion
- Allow uploads while the device is on battery while under set battery level.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**